javascript-timers/interval.js

```diff
@@ -0,0 +1,12 @@
+const head = document.querySelector('.countdown-display');
+
+const myInt = setInterval(() => {
+  if (head.textContent === '1') {
+    clearInterval(myInt);
+    setTimeout(() => {
+      head.textContent = '~ Earth Beeeelooowww Us~';
+    }, 500);
+  } else {
+    head.textContent = head.textContent -= 1;
+  }
+}, 1000);
```

Comment:
You shouldn't be using `setTimeout` & `setInterval` here. You should just be using `setInterval`. You should be calling `setInterval` outside this function, passing in what function you want to be ran. The only thing that should be happening inside this function besides updating textContent is using `clearInterval`